src/apps/components/run-header.js

```diff
@@ -44,38 +44,44 @@ class RunHeader extends React.Component {
         this.refs.Dropdown.close();
     };
 
-    _renderToggles() {
+    _renderCodeButton() {
         let showJuttleClasses = classnames('btn', 'btn-default', 'btn-code', {
             'active': this.state.showJuttle
         });
-        let showDebugClasses = classnames('btn', 'btn-default', 'btn-code', {
-            'active': this.state.showDebug
-        });
-
         let query = !this.props.bundle ? { local: true } : this.props.location.query;
-
-        var codeBtn;
-        if (this.props.runMode.path || this.props.runMode.rendezvous || this.props.runMode.local) {
-            codeBtn = (<button
-                onClick={this._toggleShowJuttle}
-                className={showJuttleClasses}>
-                <i className="fa fa-lg fa-fw fa-code"></i>
-                <div className="font-btn-name">code</div>
-            </button>);
+        let codeButtonProps = {
+            onClick: this._toggleShowJuttle,
+            className: showJuttleClasses
+        };
+        let codeIcon =  (<i className="fa fa-lg fa-fw fa-code"></i>);
+        let codeTitle = (<div className="font-btn-name">code</div>);
```

Comment:
You've probably considered it but why not put `codeTitle` and `codeIcon` together in the same definition?